.github/workflows/rebase.yaml

```diff
@@ -7,14 +7,13 @@ jobs:
     name: Rebase
     if: |
       (github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')) &&
-      (github.event.comment.author_association == 'MEMBER' || github.event.comment.user.login == github.event.pull_request.user.login)
```

Comment:
What were the values for these two fields when you triggered the workflow?

Reply:
github.event.comment.author_association = "Owner"
github.event.comment.user.login = "TiberiuGC"
github.event.pull_request.user.login = "" 😕

Reply:
> Don't we need to remove the `&&` on L9?

🙈 🙈 🙈

Reply:
So I did some digging and if the workflow is triggered by an `issue_comment` event, the PR opener should be retrieved using `github.event.issue.user.login`, instead of `github.event.pull_request.user.login` (which is used if the worflow is triggered by a `pull_request` event).

I propose to keep the condition as below:

`github.event.comment.author_association` == `OWNER` || 
`github.event.comment.author_association` == `MEMBER` || 
(`github.event.issue.user.login` == `github.event.comment.user.login`)

This way, owners and members will be able to rebase any PRs, while other type of contributors will only be able to rebase their PRs.

Reply:
tbh, I'd just remove this condition altogether but I am fine with the above suggestion 👍🏻